### ISSUE TITLE
Feature Request: Set file/folder sync priority

### ISSUE THREAD

**Issue by alexbixor** [nextcloud/desktop#723]
### Expected behaviour
It would be very useful to be able to set/change the priority of up/downloads from the client UI.
An example of this is the MEGA client, whereby it is possible, albeit manually, to move an upload to the top of the queue and vice versa.
This is particularly useful in work-flows that have varied file sizes to sync continuously, VFX is a fine example of this, where it is common to have a huge multi GB file blocking other uploads for hours, thus slowing down the whole team. 

### Actual behaviour
This is no possible at all at this point.

### Steps to reproduce
1.
2.
3.

### Client configuration
Client version:
2.X
Operating system:
Win/OSX/Linux
OS language:
Any
Qt version used by client package (Linux only, see also Settings dialog):

Client package (From Nextcloud or distro) (Linux only):

Installation path of client:

### Server configuration

Operating system:

Web server:

Database:

PHP version:

Nextcloud version:

Storage backend (external storage):

### Logs

Please use Gist (https://gist.github.com/) or a similar code paster for longer
logs.

```Template for output < 10 lines```

1. Client logfile: Output of `nextcloud --logwindow` or `nextcloud --logfile log.txt`
(On Windows using `cmd.exe`, you might need to first `cd` into the Nextcloud directory)
(See also https://docs.nextcloud.com/desktop/2.3/troubleshooting.html#log-files)

2. Web server error log:

3. Server logfile: nextcloud log (data/nextcloud.log):

**Comment by JonasDaWi** [comment]
Hi,

this would be indeed very useful, eg. also in low-bandwidth situations where you want to upload important files first.

Greetings!

**Comment by patricekaratchentzeff** [comment]
Hi,

I'm also interested, event if you can do it only server side.

Thanks,

**Comment by jmkiol** [comment]
Support for that!

**Comment by vexx32** [comment]
Would definitely love the ability to do this! Especially for syncing large folders, where I might want smaller, more numerous files synced first than for it to get stuck for a good solid hour or two syncing one large file before it even gets started on the rest of the file.

**Comment by tdamsma** [comment]
There is definitely demand for this, it was also requested (and rejected) previously: https://github.com/nextcloud/server/issues/7204

**Comment by mark3apps** [comment]
Please Make this happen!  We have TB's of data and things can really bog down when we're stuck on some large files.

**Comment by liquididea** [comment]
It would be great if you add this feature. 

As a workaround, you can setup mulitple nextcloud instances and disable the sync of the first instance when you first want to download other stuff.

**Comment by AnttiPerala** [comment]
Would really appreciate this feature.  At the moment I'm syncing large video files but would really need a small text file first.

**Comment by eekhof** [comment]
I would also be really interested in such a feature :)

**Comment by jadtl** [comment]
I'm interesting as well, it would be useful in my use-case where I would like Obsidian files to sync so I can pick them up again seamlessly between my devices, before sometimes huge files sync from other directories!

**Comment by DorianVasco** [comment]
Indeed, that would be an awesome feature! I'm also handling a lot of footage which will upload slowly. But in the time I also edit a few small text documents which should get uploaded inbetween the huge stuff.

**Comment by nikbpetrov** [comment]
+1 I got backups syncing on multiple devices (not an ideal setup, I know, but sue me) and would like to assign lowest priority to that. If I open a device I have not used for work in a while, I want critical files to be synced first.
